src/main/java/Storage.java

```diff
@@ -0,0 +1,87 @@
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+    private File folder;
```

Comment:
Try to use a more descriptive name for this, I may know it is a folder but I don't know what this folder is for